### ISSUE TITLE
Send Text Error Handling

### ISSUE THREAD

**Issue by jampy** [open-wa/wa-automate-nodejs#645]
I have a valid license (the module confirms so upon start) but when I send a message to a new number, nothing is being sent and the function returns the *string* `Not a contact`.

Shouldn't this work with a valid license?

Startup log:

```
✓ Initializing WA
┌─────────────────────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│       (index)       │                                                                    Values                                                                     │
├─────────────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│     WA_VERSION      │                                                                  '2.2027.10'                                                                  │
│       PAGE_UA       │ 'WhatsApp/2.2019.8 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36' │
│ WA_AUTOMATE_VERSION │                                                                   '1.9.98'                                                                    │
│   BROWSER_VERSION   │                                                        'HeadlessChrome/81.0.4044.138'                                                         │
└─────────────────────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
✓ Authenticated
✓ WAPI Reinjected
✓ Client is ready
✓ License Valid
✓ 🚀 Ready!

```

BTW, what is the correct method to check if `sendText()` was successfull? It returned `false` in earlier versions but now it appears to return a string in any case... :-(

**Comment by smashah** [comment]
@jampy can you dm me on discord please to fix this ASAP

**Comment by jampy** [comment]
@smashah what's your nickname there? "SHAH"?

**Comment by smashah** [comment]
Work to be done:

- [ ] Make sure the errors from the page get logged and any unsuccessful attempts to send a message return `false` instead of the error message (`Not a contact`)

**Comment by smashah** [comment]
Implemented in v1.9.99 - errors will be logged with `console.error` and the function will return false as was done previously
